execute/executetest/table.go

```diff
@@ -480,6 +481,88 @@ func NormalizeTables(bs []*Table) {
 	for _, b := range bs {
 		b.Key()
 	}
+	bs = sortByGroupKey(bs)
+}
+
+// sortByGroupKey uses recursive quicksort to sort the tables by group key
+// in alphabetical order
+func sortByGroupKey(tables []*Table) []*Table {
+	if len(tables) < 2 {
+		return tables
+	}
+
+	for _, table := range tables {
+		// somehow this line makes the test pass
+		// no idea why
+		_ = table.Key().String()
+	}
```

Comment:
I'm not sure why, but removing this for loop causes `TestNormalizeTables()` to fail. Any ideas?